Backend/routes/profile.py

```
@@ -0,0 +1,96 @@
+        db.commit()
+
+        return jsonify({"status": "success", "message": "Profile set successfully with due date","dueDate": due_date}), 200
+    
+    except sqlite3.OperationalError as error:
+        return jsonify({"error": str(error)}), 500
+
+@profile_bp.route('/get_profile', methods=['GET'])
+def get_profile():
+    db = open_db()
+
+    try: 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)